lib/puppet/type/syslog_settings.rb

```diff
@@ -115,6 +150,26 @@
       vrf: {
         type:   'Optional[Array[String]]',
         desc:   'The VRF associated with source_interface (array of strings for multiple).'
+      },
+      logfile_severity_level: {
+        type:   'Optional[Integer[0,7]]',
+        desc:   'Logfile severity level [0-7]'
+      },
+      logfile_name: {
+        type:   'Optional[String]',
+        desc:   "Logfile file name to use or 'unset'"
+      },
+      logfile_size: {
+        type:   'Optional[Integer[4096, 2147483647]]',
```

Comment:
I believe we wanted to drop the range checking here as well.

Reply:
@shermdog Apologies I had the changes sitting locally and forgot to push them up! Should be there now :)